### PR TITLE
iamsurajdc/brandPage

### PR DESCRIPTION
**Pull request by iamsurajdc**
**Description**

This PR fixes [#712](https://github.com/layer5io/layer5/issues/712)

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
@iamsurajdc Nice initial work. 
- The gray background is not looking good. Keep it white.
- You have used images for different color pallets. We will want them to be a div filled with the required colors

**Comment by iamsurajdc** [comment]
@Tanuj22 Thank you for the compliment,
If there would be white `bg`, then It'll lose the differentiating container panes along with the `border-radius` format for respective sections, Are you sure to get rid of grey with white?

**Comment by Tanuj22** [comment]
> @Tanuj22 Thank you for the compliment,
> If there would be white `bg`, then It'll lose the differentiating container panes along with the `border-radius` format for respective sections, Are you sure to get rid of grey with white?

I don't think the gray color is looking nice here. @Nikhil-Ladha what you think?

**Comment by Nikhil-Ladha** [comment]
I agree with @Tanuj22 gray looks a bit odd and doesn't quite goes with the brand colors (which ironically you are working upon 😅).

**Comment by Nikhil-Ladha** [comment]
> @Tanuj22 Thank you for the compliment,
> If there would be white `bg`, then It'll lose the differentiating container panes along with the `border-radius` format for respective sections, Are you sure to get rid of grey with white?

Well, your point is valid that we won't get the border radius effect just by changing the bg color to white, but adding a box shadow to it will make it look evident. Give that a shot.

**Comment by leecalcote** [comment]
@iamsurajdc there is a merge conflict that you will want to take a look at.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Great work here @iamsurajdc . A couple of changes needed -

- In the mobile view, the color and the grayscale headings are together. Rather they should be above each of there corresponding parts.
Screenshot from 2020-10-03 14-21-27

- Merge `layer5-ng` branch into your branch. Then use the `Button` component already available. The various classes like `waves` etc. can then be removed safely.

- Create a separate file for each of the sections of `Layer5`, `Mystery`,  `ImageHub` etc. This will make it easier to manage as we add more sections in the future. 

P.S. I know these may seem like many changes but this will make the code more modular and cleaner. If you are stuck somewhere or need more clarification please ask :)

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Really awesome work @iamsurajdc. Three small changes and we are good to go. 

- The color pallets should be below there respective headings. Keep the color and grayscale side by side. (This should also be the case in mobile view) Check the current site for reference. 
Screenshot from 2020-10-05 13-02-18

- Align this to the pallets in meshery and SMP to center in mobile view. 
Screenshot from 2020-10-05 13-06-09

- Keep the contact us part below the heading in the mobile view. 
Screenshot from 2020-10-05 13-01-32

**Comment by leecalcote** [comment]
Looking good, @iamsurajdc. Nearly at the finish line. :)

**Comment by Nikhil-Ladha** [comment]
A nice improvement here, few more improvements needed:
- The section header should have a bigger font size and should be properly left-aligned similar to other sub-headings.
- The grayscale and the color, colors are getting mixed up as we make the view responsive and move to a mobile view or tablet view. Please fix that.

File: src/sections/Brand/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { Link } from "gatsby"
```
Comment (Nikhil-Ladha) [inline src/sections/Brand/index.js:2]: Also, please fix these warnings. This one and 2-3 below it, only on this file.
The warning for unused variables.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
@iamsurajdc  ^^

**Comment by Tanuj22** [comment]
@Nikhil-Ladha actually @iamsurajdc asked to move forward with this PR as he is unable to make the changes. We can create the issue for the changes needed.

**Comment by Jashpatel1** [comment]
> @Nikhil-Ladha actually @iamsurajdc asked to move forward with this PR as he is unable to make the changes. We can create the issue for the changes needed.

Yes, I think we can move with these changes and open up another issue for updates, if it suits well @Nikhil-Ladha

**Comment by Nikhil-Ladha** [comment]
Ohh, ok ok. No worries.
Good that you spoke and confronted that @iamsurajdc :+1: 
Let's move forward over here...some more updates can be made in future PRs!

**Review by Nikhil-Ladha — APPROVED** [review]
Good work here @iamsurajdc :sunglasses:

**Comment by Jashpatel1** [comment]
Great work here @iamsurajdc with sheer patience.

**Review by Tanuj22 — APPROVED** [review]
Thank you @iamsurajdc for this great work :). Can you create an issue for all the changes that are left to be done?

**Comment by leecalcote** [comment]
@iamsurajdc, I’d say this is worth a GitHub star on the Meshery project. Star your work. :) good of you to deliver this first PR. 👍

**Comment by iamsurajdc** [comment]
> Thank you @iamsurajdc for this great work :). Can you create an issue for all the changes that are left to be done?

I will

**Pull request merged**